harerajo/node-practice
# 34-testing-all-endpoints

#### What does this PR do?

- Implements rest APIs tessting
#### Description of Task to be completed? 

- User can be able to add a todo task

#### How should this be manually tested?

- Clone the repo

- checkout to branche ``fit-testing``

- Run `npm install` to install modules

- Run `npm  run start` to start the development server

- run `npm test` to see the tests coverage

#### Any background context you want to provide?

- N/A
#### What are the relevant Pivotal Tracker/trello stories?

-https://trello.com/c/h7sd7BjG

#### Screenshots (if appropriate)
- N/A

#### Questions:

-N/A
